core/lib/kv/index.ts

```diff
@@ -20,24 +24,42 @@ class KV<Adapter extends KvAdapter> implements KvAdapter {
   }
 
   async get<Data>(key: string) {
-    const kv = await this.getKv();
-    const fullKey = `${this.namespace}_${key}`;
-
-    const value = await kv.get<Data>(fullKey);
```

Comment:
If I'm reading correctly this removes calls to the individual adapters' `get` methods entirely in favor of `mget`. Are we sure we want that? And if so, should we remove those methods?

Reply:
Good point, removed unused methods